.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Should we run tests before publishing? Could you describe process of publishing the gem?

Reply:
We can add it as step before publishing the gem